workflow-templates/1gp-ci.yml

```
@@ -56,30 +56,64 @@ on:
 
       auto-promote:
         description: Whether to auto promote the package
-        value: "false"
+        value: 'false'
 
 permissions:
   id-token: write # This is required for requesting the ID token for AWS actions
   contents: read # This is required for actions/checkout
 
 concurrency:
-  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
   cancel-in-progress: ${{ github.ref != 'refs/heads/release' }}
 
 env:
-  DD_AGENT_MAJOR_VERSION: "7"
-  DD_API_KEY: "${{ secrets.DD_API_KEY }}"
-  DD_APP_KEY: "${{ secrets.DD_APP_KEY }}"
+  DD_AGENT_MAJOR_VERSION: '7'
+  DD_API_KEY: '${{ secrets.DD_API_KEY }}'
+  DD_APP_KEY: '${{ secrets.DD_APP_KEY }}'
   DD_SITE: datadoghq.com
-  DEVOPS_SCRIPTS_PATH: "./devops-utility-belt/scripts"
+  DEVOPS_SCRIPTS_PATH: './devops-utility-belt/scripts'
   GIT_BASE_BRANCH: ${{ github.base_ref }}
   GIT_BRANCH: ${{ github.ref_name }}
-  GITHUB_TOKEN: "${{ secrets.CI_USER_REPO_ACCESS_TOKEN }}"
+  GITHUB_TOKEN: '${{ secrets.CI_USER_REPO_ACCESS_TOKEN }}'
   RELEASE_FULLNAME: nCino Companies House (UK)
   REPO_NAME: ${{ github.event.repository.name }}
+  NAMESPACE: << REPLACE WITH PROJECT NAMESPACE >>
 
 jobs:
+  Path-Filtering:
+    outputs:
+      apex_changed:: ${{ steps.path-filtering.outputs.apex_changed }}
+      lwc_changed: ${{ steps.path-filtering.outputs.apex_changed }}
+      markdown_changed:: ${{ steps.path-filtering.outputs.markdown_changed }}
+      non_markdown_changed:: ${{ steps.path-filtering.outputs.non-markdown_changed }}
+    runs-on: ubuntu-latest
+    steps:
+      # Uses the action by @dorny to filter paths
+      # More info can be found at https://github.com/dorny/paths-filter
+      - name: Run Path Filtering
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36
```

Could you leave a comment next to this with a version number that this commit relates to? Something like this:

```suggestion
        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36  // version 1.2.3
```

---

10-4